src/Middleware.php

```diff
@@ -91,12 +91,13 @@ public function __invoke(Request $request, Response $response, callable $out = n
      *
      * A handler can be any callable, or an object with a handle() method.
      *
-     * Handlers with arity >= 4 are considered error handlers, and will
+     * Handlers with parity >= 4 are considered error handlers, and will
```

Comment:
Actually, "arity" is the correct term here; see https://www.google.com/search?q=define%3Aarity